lib/wagn/generators/wagn/templates/config/application.rb

```diff
@@ -16,6 +16,10 @@ class Application < Wagn::Application
   
     
     # Email configuration
+
+    if !Rails.env.production? && Object.const_defined?( :JasmineRails )
+        config.assets.enabled = true
+    end
```

Comment:
We don't want this here in the templates.  Perhaps in the environment files?  Do we need it in development or only test?